### PR TITLE
Drup8 1161

### PR DESCRIPTION
**Pull request by mechanoid23**

### CODE REVIEW

**Comment by mechanoid23** [comment]
Please look at the config items, particularly search path for GCSE.

File: osu_standard.install [hunk 1]
```diff
@@ -47,6 +47,31 @@ function osu_standard_install($is_syncing) {
...
+  $search404_settings->set('search404_disable_error_message', TRUE);
+  $search404_settings->set('search404_use_search_engine', TRUE);
+  $search404_settings->set('search404_do_google_cse', TRUE);
+  $search404_settings->save();
+}
+
+/**
+ * Install Search 404 module.
+ */
+function osu_standard_update_9041(&$sandbox) {
```
Comment (emerham) [inline osu_standard.install]: These two update numbers are the same, it you want to run them in two separate updates you'll need to bump the other one that sets the configuration. You can also just do both of these in one update hook.

File: osu_standard.install [hunk 2]
```diff
@@ -47,6 +47,31 @@ function osu_standard_install($is_syncing) {
...
+  \Drupal::service('module_installer')->install([
+    'search404',
+  ], TRUE);
+  return t('Modules xmlsitemap_custom, xmlsitemap_engines are enabled.');
```
Comment (emerham) [inline osu_standard.install]: Update return statement to say what modules you've enabled.

**Review by emerham — CHANGES_REQUESTED** [review]
some minor changes

File: osu_standard.install (hunk 1 above)
Comment (mechanoid23) [reply osu_standard.install]: How to tell me I didn't test my code without telling me I didn't test my code?
Single update, okay. Thought to keep them as separate hooks.

File: osu_standard.install (hunk 2 above)
Comment (mechanoid23) [reply osu_standard.install]: Statement updated.

**Review by Ruefa — APPROVED** [review]

**Pull request merged**